cluster/cluster.go

```diff
@@ -122,7 +122,23 @@ func GetMemberPIDs(kind string) actor.PIDSet {
 	return pids
 }
 
-// RemoveCache at PidCache
+// GetMemberAddresses returns the list of addresses of all currently known cluster members.
+func GetMemberAddresses() []string {
```

Comment:
Im curious why this function is here if its not being used anywhere else in the PR

Reply:
Good catch, this is used inside the `connectionDecider` function. I forgot to put it in here - but you can find it's usage in your PR #334. Here the excerpt:

```go
remotingOption = append(remotingOption, remote.WithConnectionDecider(func(address string) bool {
	adrList := cluster.GetMemberAddresses()
	for _, v := range adrList {
		if v == address {
			return true
		}
	}
	log.Infof("Address %s is no known member. Deciding not to connect", address)
	return false
}))
```